client/src/components/ForumPostContainer/ForumPostContainer.js

```diff
@@ -22,28 +23,27 @@ export default function ForumPostContainer(props) {
             post={props.forumPost.post}
           />
         ) : (
-          <ForumPost />
-        )}
+            <ForumPost />
+          )}
       </div>
       <div>
-        {props.comments ? (
-          props.comments.map((comment, i) => {
+        {!comments ? "" : (
+          comments.map((comment, i) => {
             return (
               <Comment
-                name={comment.name}
-                likes={comment.likes}
-                replies={comment.replies}
-                image= {comment.image}
-                date={comment.date}
-                time={comment.time}
-                body={comment.body}
-                key={i}
+                {...comment}
+              // name={comment.name}
+              // likes={comment.likes}
+              // replies={comment.replies}
+              // image={comment.image}
+              // date={comment.date}
+              // time={comment.time}
+              // body={comment.body}
+              // key={i}
```

Comment:
This is better but please delete any commented out code like this

Reply:
As a general rule, you never want to submit a PR with commented out code, `console.log` statements, or errors

Reply:
Done